README.md

```diff
@@ -1,2 +1,63 @@
 # fsr
-FSR code used for dance pads
+A complete software package for FSR dance pads.
+
+## Features
+- React web UI to customize sensitivity 
+- Profiles & persistence
+- Light support
+
+## Screenshots
+<img src="./img/fsr2.gif">
+
+<img src="./img/fsr1.gif">
+
+
+## Requirements
+- A [Teensy](https://www.pjrc.com/store/index.html)
+- Python 3.6+
+    - virtualenv
+- Node 12+
+  - yarn or npm
+
+## Hardware setup
+Follow a guide like [fsr-pad-guide](https://github.com/Sereni/fsr-pad-guide) or [fsr](https://github.com/vlnguyen/itg-fsr/tree/master/fsr) to setup your Arduino/Teensy with FSRs.
+
+## Firmware setup
+1. Install [Arduino IDE](https://www.arduino.cc/en/software) (skip this if you're using OSX as it's included in Teensyduino)
+1. Install [Teensyduino](https://www.pjrc.com/teensy/td_download.html) and get it connected to your Teensy and able to push firmware via Arduino IDE
+1. In Arduinio IDE, set the `Tools` > `USB Type` to `Serial + Keyboard + Mouse + Joystick`
+1. In Arduinio IDE, set the `Tools` > `Board` to `Teensy 4.0`
```

Comment:
Probably change this to make it more generic than "Teensy 4.0" something like "to your microcontroller"